test/backports/ValueIdentityTest.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright 2025, Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+
+#include <gtest/gtest.h>
+
+#include "util/ValueIdentity.h"
+
+using namespace ad_utility;
+using namespace ad_utility::use_value_identity;
+
+TEST(ApplyAsValueIdentity, FunctorInvocation) {
+  constexpr static auto functor = [](auto... compile_time_values,
```

Comment:
Interesting that this compiles, I'll have a look at it myself:) (with the two variadics without any means to distinguish between them).